src/utils/prepareSwapTransactions.test.ts

```diff
@@ -206,50 +205,11 @@ describe('prepareSwapTransactions', () => {
       to: '0x12345678',
       data: '0xswapdata',
       value: 111n,
-      gas: 12345n,
+      gas: 14042n,
       estimatedGasUse: 12211n,
     })
   })
 
-  it('uses default gas for postHook', async () => {
```

Comment:
this test was redundant, post hooks always use the default gas and this is tested in other tests (left over cleanup from #582)